LayoutTests/inspector/css/getMatchedStylesForNodeNestingStyleGrouping-expected.txt

```diff
@@ -31,7 +31,7 @@ PASS: Rule should have no groupings.
 -- Running test case: CSS.getMatchedStyleForNode.NestingStyleGrouping.NestedRulePropertyWithImplicitAmpersand
 PASS: Should have 4 authored rules.
 - Testing rule #0
-PASS: Selector text should be ".innerA".
+PASS: Selector text should be "& .innerA".
```

Comment:
This doesn't seem right. Web Inspector should show _authored styles_ and _not_ the serialized style, which in this case wouldn't include the implicit ampersand.